xml/admin_gui_dashboard.xml

```diff
@@ -2287,6 +2287,89 @@ Oct 04 07:48:53 doc-ses6min4 systemd[1]: Started Ceph rbd mirror daemon.
 &prompt.cephuser;ceph dashboard create-self-signed-cert
 </screen>
    </sect3>
+   <sect3 xml:id="self-sign-certificates-openssl">
+     <title>Self-signed or Trusted Third-party Certificate with OpenSSL</title>
+     <para>
+       OpenSSL is an open-source command line tool that is commonly used
+       to generate private keys, create CSRs, install your SSL/TLS
+       certificate, and identify certificate information. The following
+       instructions illustrate to generate a self-signed or trusted third-party
+       certificate using OpenSSL:
+     </para>
+<!-- Source: https://www.suse.com/support/kb/doc/?id=000019241 -->
+     <procedure>
+       <step>
+         <para>
+           Generate a Private Key:
+         </para>
+<screen>
+openssl genrsa -des3 -out server.key 2048
+</screen>
+         <para>
+           Type the pass phrase to protect the key.
+         </para>
+       </step>
+       <step>
+         <para>
+           Generate a Certificate Signing Request:
+         </para>
+<screen>
+openssl req -new -key server.key -out server.csr
+</screen>
+         <para>
+           Enter the passphrase, and fill in the <literal>Country Name</literal>,
+           <literal>State or Province Name</literal>, <literal>Locality Name</literal>, <literal>Organization Name</literal>,
+           <literal>Organizational Unit Name</literal>, <literal>Common Name</literal>,
+           <literal>Email Address</literal>.
+         </para>
+         <note>
+           <para>
+             The <literal>Common Name</literal> should be the DNS name of the
```

Comment:
```suggestion
             The <literal>Common Name</literal> should be the FQDN name of the
```

Reply:
So, I'm not going to change this immediately because this is a copy of a SUSE KB file: https://www.suse.com/support/kb/doc/?id=000019241 :thinking:

Reply:
I (mostly!) agree with @cwickert on this -- I think that KB article is badly phrased. 

For me it's a warning sign whenever a word would be duplicated:

> should be the domain name service name of the

or 

> should be the fully-qualified domain name name of the

So I would write:

The Common Name should be the FQDN of the...

Reply:
Alright, I'll roll with it :)